se_shared/include/se/path_planning/candidate_view.hpp

```diff
@@ -343,7 +343,13 @@ std::pair<double, float> CandidateView<T>::getInformationGain(const Volume<T> &v
         }
       }
     }
+  } else{
+    // TODO 0.4 is set fix in ray_iterator
+    float num_it = (tfar-0.4)/step;
```

Comment:
0.4 is probably the nearPlane, defined in `se_denseslam/include/se/constant_parameters.h`

Reply:
Also float can be made const